pipenv/cli.py

```diff
@@ -37,7 +37,7 @@
 click_completion.init()
 
 # No color mode, for unfun people.
-if 'PIPENV_COLORBLIND' in os.environ:
+if os.environ.get('PIPENV_COLORBLIND') == '1':
```

Comment:
For sake of being complete maybe support other known 1-ish values ( on, yes, 1, true).